source/server/active_listener_base.h

```diff
@@ -15,6 +15,8 @@ namespace Server {
   COUNTER(downstream_cx_overload_reject)                                                           \
   COUNTER(downstream_global_cx_overflow)                                                           \
   COUNTER(downstream_pre_cx_timeout)                                                               \
+  COUNTER(downstream_peek_remote_close)                                                            \
+  COUNTER(downstream_peek_error)                                                                   \
```

Comment:
Similar: `downstream_pre_cx_read_error` (not peek; I think that's too low level of a detail) or `downstream_listener_filter_read_error`.